js/tests/unit/dom/data.spec.js

```diff
@@ -102,5 +90,17 @@ describe('Data', () => {
     expect(console.error).toHaveBeenCalled()
     expect(Data.get(div, UNKNOWN_KEY)).toBeNull()
   })
+
+  it('should console.error a message if "set" is for the same element', () => {
+    console.error = jasmine.createSpy('console.error')
+
+    const data = { ...TEST_DATA }
+
+    Data.set(div, TEST_KEY, data)
+    Data.set(div, TEST_KEY, data)
+
+    expect(console.error).toHaveBeenCalled()
```

Comment:
Could we also check if the message includes/starts with `Bootstrap doesn't allow more than one instance per element`?

Reply:
ready ;)